packages/app/src/components-styled/content-header/metadata.tsx

```diff
@@ -66,7 +66,7 @@ export function Metadata(props: MetadataProps) {
         label={text.source}
         accessibilityText={replaceVariablesInText(
           siteText.accessibility.link_source,
-          { title }
+          { accessibilitySubject, source: 'test' }
```

Comment:
nit;
```suggestion
          { subject: accessibilitySubject, source: 'test' }
```
Because within the context of lokalize I don't think it has value to have this variable name spelled out with accessibility. Subject is clear enough in that text.